presto-hive/src/main/java/io/prestosql/plugin/hive/security/LegacyAccessControl.java

```diff
@@ -247,6 +247,16 @@ public void checkCanSetCatalogSessionProperty(ConnectorSecurityContext context,
     {
     }
 
+    @Override
+    public void checkCanGrantSchemaPrivilege(ConnectorSecurityContext context, Privilege privilege, String schemaName, PrestoPrincipal grantee, boolean grantOption)
```

Comment:
I think this should accept a `Set<Privilege>` like `Metadata::grant*`. No change request, just for consideration for follow up because we would need to change check for tables as well.